From ce433f52461ad9aa3839cdf01fda26f15262f8ab Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Thu, 28 Aug 2014 03:04:41 -0400
Subject: [acpi] Use EXPORT_SYMBOL() for acpi_bus_get_device()

Message-id: <20140828030441.2117.30690.stgit@gir.stowe>
Patchwork-id: 89299
O-Subject: [RHEL7.1 PATCH 11/30] ACPI: Use EXPORT_SYMBOL() for acpi_bus_get_device()
Bugzilla: 1128632
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1128632
Upstream Status: 6585925b62fa4dd2f4aecf634b0f8956577aa981

commit 6585925b62fa4dd2f4aecf634b0f8956577aa981
Author: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Date:   Tue Oct 1 23:02:43 2013 +0200

    ACPI: Use EXPORT_SYMBOL() for acpi_bus_get_device()

    Commit caf5c03f (ACPI: Move acpi_bus_get_device() from bus.c to
    scan.c) caused acpi_bus_get_device() to be exported using
    EXPORT_SYMBOL_GPL(), but that broke some binary drivers in
    existence, so revert that change.

    Reported-by: Peter Hurley <peter@hurleysoftware.com>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/acpi/scan.c b/drivers/acpi/scan.c
index e76b523..baf5100 100644
--- a/drivers/acpi/scan.c
+++ b/drivers/acpi/scan.c
@@ -927,7 +927,7 @@ int acpi_bus_get_device(acpi_handle handle, struct acpi_device **device)
  }
  return 0;
 }
-EXPORT_SYMBOL_GPL(acpi_bus_get_device);
+EXPORT_SYMBOL(acpi_bus_get_device);
 
 int acpi_device_add(struct acpi_device *device,
       void (*release)(struct device *))
-- 
1.7.1